### PR TITLE
new readme, included song spec

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
@jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
+## Overview
+
+This is your first introduction to the "Playlister" domain model. In this first iteration, we'll be building a class for `Song`, `Artist` and `Genre`. A song will will belong to an artist and an artist will have many songs. A song belongs to a genre and a genre belongs to a song. So, an Artist has many genres *through their songs*.
 
-### Skills: Object Oriented Programming, Test Driven Development
 
 ## Instructions
 
-Fork this repository, clone it, and make the RSpec tests pass.
+For the most part, this is a test-driven lab. Follow the test output to define you classes with the appropriate attributes. The guidelines below are meant to help you build the associations. Read them through first, then proceed through the test suite by running `rspec --fail-fast`. Refer back to these guidelines to help you pass tests related to associations.
```
Comment (jmburges) [inline README.md]: drop the mention of `rspec` as much as possible we want them using learn!

File: README.md [hunk 2]
```diff
@@ -1,20 +1,44 @@
...
+  * At what point in time does a song become associated with an artist? In the real world, that happens when an artist writes a song. So, the `add_song` instance method on the `Artist` class should not only push a song into that artists `songs` array but set the song's `artist` attribute equal to the artist that you are operating on. 
+* `Genre`s should have many songs.
+  * The `Genre` class should have an `attr_accessor`, `songs`.
+  * The `Genre` class should initialize with an empty array, `@songs`.
+* An instance of `Song` should belong to a genre. At what point do a genre and a song get associated to one another? This decision is a little more flexible than our decision to associate artists and songs in an artist's `add_song` method. For the purposes of passing the tests, however, we'll say that a genre and a song become associated to one another **when a song's genre is set**. 
+  * Write out the `.genre=(genre)` setter method in the song class explicity. In it, set the `@genre` variable equal equal to the genre that will be passed in as an argument. **Also in this method, push the song instance (i.e. `self`) into that genre's `.songs` array**.
```
Comment (jmburges) [inline README.md]: use either the `#` like below, or the `.`. I think the `#` makes the most sense

File: spec/song_spec.rb [hunk 3]
```diff
@@ -1,17 +1,25 @@
+require 'pry'
```
Comment (jmburges) [inline spec/song_spec.rb]: no require pry in specs

File: spec/song_spec.rb [hunk 4]
```diff
@@ -1,17 +1,25 @@
...
+     song.name = "Twinkle Twinkle Little Star"
+     expect(song.name).to eq("Twinkle Twinkle Little Star")
   end
 
-  xit "can have a genre" do
-     # implement this spec
+  it "can have a genre" do
+     hip_hop = Genre.new
+     song.genre = hip_hop
+     binding.pry
```
Comment (jmburges) [inline spec/song_spec.rb]: pry!

**Comment by jmburges** [comment]
few issues and :shipit:

File: README.md (hunk 2 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed.

File: spec/song_spec.rb (hunk 3 above)
Comment (SophieDeBenedetto) [inline spec/song_spec.rb]: fixed.

File: spec/song_spec.rb (hunk 4 above)
Comment (SophieDeBenedetto) [inline spec/song_spec.rb]: fixed.

File: README.md (hunk 1 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed.

**Comment by SophieDeBenedetto** [comment]
@jmburges, addressed all comments.

**Pull request merged**